src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingUploadConfirmSteps.java

```diff
@@ -420,11 +358,8 @@ public void inspectSurveillanceDetails(final String chplId) {
 
     /**
      * Confirm uploaded surveillance activity.
-     * 
-     * @param survChplId
-     *            is chpl id of listing for surveillance activity
-     * @throws Exception
-     *             if there is an exception
+     * @param survChplId is chpl id of listing for surveillance activity
```

Comment:
One of the checkstyle rules we've discarded is having the javadocs. Unless you think this is useful information, I'm on board with just removing them, especially for super small methods like these

Reply:
sounds good, will update with other checkstyle fixes in next PR